packages/create-llama/templates/components/sample-projects/llamapack/README.md

```diff
@@ -0,0 +1,14 @@
+TODO: Add description of this project here.
```

Comment:
how about copying the README from the llamapack? and appending these install instructions?